### PR TITLE
Update printPlanWithStats documentation

### PR DESCRIPTION
**Pull request by majetideepak**

### CODE REVIEW

File: velox/docs/develop/debugging/print-plan-with-stats.rst [hunk 1]
```diff
@@ -226,18 +230,18 @@ groups were skipped via stats-based pruning.
...
-            numRamRead                 sum: 0, count: 1, min: 0, max: 0
-            numPrefetch                sum: 2, count: 1, min: 2, max: 2
-            prefetchBytes              sum: 31880, count: 1, min: 31880, max: 31880
-            localReadBytes             sum: 0, count: 1, min: 0, max: 0
-            skippedSplits              sum: 0, count: 1, min: 0, max: 0
-            storageReadBytes           sum: 63010, count: 1, min: 63010, max: 63010
-            numStorageRead             sum: 7, count: 1, min: 7, max: 7
-            skippedSplitBytes          sum: 0, count: 1, min: 0, max: 0
-            numLocalRead               sum: 0, count: 1, min: 0, max: 0
+   -> TableScan[Table = Projection]
```
Comment (mbasmanova) [inline velox/docs/develop/debugging/print-plan-with-stats.rst:233]: It is strange to see `Table = Projection` in these examples. Perhaps, change to something more realistic, e.g. Table = Orders.

**Review by mbasmanova — COMMENTED** [review]
@majetideepak Deepak, thank you for updating the docs to catch up with recent improvements.

File: velox/docs/develop/debugging/print-plan-with-stats.rst (hunk 1 above)
Comment (majetideepak) [reply velox/docs/develop/debugging/print-plan-with-stats.rst]: Done!

**Review by mbasmanova — APPROVED** [review]
@majetideepak Thank you, Deepak.

**Pull request closed without merging**